modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/model/ApplicationModel.java

```diff
@@ -602,49 +805,6 @@ private void executeOnComponentTree(final ComponentModel component, final Consum
     task.accept(component);
   }
 
-  private List<ComponentModel> extractComponentDefinitionModel(List<ConfigLine> configLines, String configFileName) {
```

Comment:
There seems that part of the logic of this method is missing. Just check that the same behaviour remains